problems/3.conditionals.js

```diff
@@ -2,25 +2,40 @@
 * @param {number} temp - The current temperature
 * @returns {string} - A message about the temperature
 */
-function temperatureMessage() {
+function temperatureMessage(temp) {
+  let temp = 32
+  let i = 0 
+  if (i <= temp) {
+    console.log("It's freezing!")
+  } else {
+    console.log ("It's alright")
+  }
 
 }
 
 /** Returns "It's freezing!" if the temperature is below 32, and "It's alright" otherwise. The implementation must use a ternary.
 * @param {number} temp - The current temperatur
 * @returns {string} - A message about the temperature
 */
-function temperatureMessageUsingTernary() {
-
+function temperatureMessageUsingTernary(temp) {
+let temp = 32 
+let i = num > 32
+return i <= temp ? "It's freezing!" : "It's alright"
 }
 
 
 /** Returns "You are a senior" if the age is 65 or older, "You are an adult" if the age is 18 or older, and "You are a minor" otherwise.
 * @param {number} age - The age of the user
 * @returns {string} - A message about their age
 */
```

Comment:
This looks good. Not sure where the dissconnect is happening between the temp questions and the age question. They should be seen as the same. Receiving an argument and making a judgment based on what they receive.